activerecord/test/cases/relation_test.rb

```diff
@@ -180,14 +180,14 @@ def test_references_values_dont_duplicate
     test "merging a hash interpolates conditions" do
       klass = Class.new(FakeKlass) do
         def self.sanitize_sql(args)
-          raise unless args == ["foo = ?", "bar"]
-          "foo = bar"
+          raise unless args == ["foo = ? AND 1 = 1", "bar"]
```

Comment:
I will need to go through all of the existing tests that use where strings with binds to make sure they are doing what they are supposed to. `AND 1 = 1` was just a quick fix to get this test passing. I will come up with something better.